modules/test-utils/src/generate-layer-tests.js

```diff
@@ -146,7 +146,7 @@ function makeAltPropTestCase({propName, propTypes, defaultProps, sampleProps, as
       ];
 
     case 'accessor': {
-      if (typeof defaultProps[propName] === 'function') {
+      if (typeof defaultProps[propName] === 'function' || propDef.optional) {
```

Comment:
As some of the accessors in the GeoJsonLayer don't always get passed on to a sublayer (e.g. `getTextBackground` when `pointType` is not `'text'`) the updateTriggers were not getting triggered, nor were the accessors getting called. This seems like the correct behavior, so I came up with this approach to fix the tests